library/xml-model/src/test/java/org/cru/godtools/xml/model/ManifestTest.kt

```diff
@@ -3,32 +3,63 @@ package org.cru.godtools.xml.model
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import java.util.Locale
 import org.cru.godtools.xml.util.getXmlParserForResource
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.empty
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
 import org.junit.Test
 import org.junit.runner.RunWith
 
-@RunWith(AndroidJUnit4::class)
 class ManifestTest {
     @Test
-    fun verifyParseEmptyManifest() {
+    fun testManifestTypeParsing() {
+        assertNull(Manifest.Type.parseOrNull(null))
+        assertEquals(Manifest.Type.ARTICLE, Manifest.Type.parseOrNull("article"))
+        assertEquals(Manifest.Type.LESSON, Manifest.Type.parseOrNull("lesson"))
+        assertEquals(Manifest.Type.TRACT, Manifest.Type.parseOrNull("tract"))
+        assertEquals(Manifest.Type.UNKNOWN, Manifest.Type.parseOrNull("nasldkja"))
```

Comment:
How do we handle parsing the unknown case?

Reply:
the `UNKNOWN` type is meant to reflect future tool types. basically when `Manifest.Type.parseOrNull()` is called with a type it doesn't recognize, it's meant to return `UNKNOWN`. This allows us to write logic that can gracefully handle future manifests we don't know about yet.

Reply:
I typed some random characters that will probably never be a valid tool type to test out the `UNKNOWN` state